crypto/ec/ecp_mont.c

```diff
@@ -247,6 +247,8 @@ int ec_GFp_mont_group_set_curve(EC_GROUP *group, const BIGNUM *p,
         BN_CTX_free(new_ctx);
     if (mont != NULL)
         BN_MONT_CTX_free(mont);
+    if (one != NULL)
```

Comment:
FYI, we did change the XXX_free routines to ignore NULL in 1.0.2 to make cherry-picking fixes easier.  So this if isn't needed.  But the one above has it for "mont" so I guess parallel construction is better.